cam0:
  cam_overlaps: []
  camera_model: omni
  distortion_coeffs: [-0.14434091609187874, 0.09199562555940563, 0.004415161735962297, 0.0002778810780371568]
  distortion_model: radtan
  intrinsics: [1.8991726492593852, 1224.8320104270672, 1222.6539209334694, 721.5159515600028, 713.817280940101]
  resolution: [1440, 1440]
  rostopic: /kodak/image_raw
